hs_core/hydroshare/hs_bagit.py

```diff
@@ -29,55 +27,47 @@ def delete_bag(resource):
     :param resource: the resource to delete the bag for.
     :return: none
     """
-    res_id = resource.short_id
-
-    res_path = resource.root_path
-    bagname = 'bags/{res_id}.zip'.format(res_id=res_id)
     istorage = resource.get_irods_storage()
-    if resource.resource_federation_path:
-        bagname = '{}/{}'.format(resource.resource_federation_path, bagname)
 
     # delete resource directory first to remove all generated bag-related files for the resource
-    if istorage.exists(res_path):
-        istorage.delete(res_path)
+    # TODO: Is this correct? Is it really necessary? It seems this deletes all content!
+    if istorage.exists(resource.root_path):
+        istorage.delete(resource.root_path)
```

Comment:
This needs to be renamed. I would assume that it deletes the bag and not the resource.